cirq/protocols/circuit_diagram_info.py

```diff
@@ -233,3 +247,7 @@ def circuit_diagram_info(val: Any,
     raise TypeError("object of type '{}' does have a _circuit_diagram_info_ "
                     "method, but it returned NotImplemented.".format(type(val)))
 # pylint: enable=function-redefined
+
+
+OPERATION_DIAGRAM_INFO_GETTER = (Callable[
```

Comment:
This isn't a protocol. It's not a methods or set of methods that, if a class implements, gives it new functionality in the eyes of other parts of the code base.

This is a short name for a specific type of method. I wouldn't even give this a name, I'd just use it directly it in signatures.

Reply:
Yea, this was just meant as a macro for concision. By "protocol style", I didn't mean that I introduced a new protocol, but more simply that I got rid of all the classes and used a functional style like that of, say, `decompose`.

Removed.